Feature/checkout (#3)
SwedbankPay/swedbank-pay-sdk-dotnet: README.md

| Build server                | Platform     | Status                                            |
|-----------------------------|--------------|---------------------------------------------------|
| Azure DevOps                | Windows      | [![Build Status][azdo-badge]][azdo-build]         |

## About

> `SwedbankPay.Sdk` is a `netstandard2` library to talk to SwedbankPay direct REST APIs.

Reviewer: ```suggestion
`SwedbankPay.Sdk` is a `netstandard2` library that allows you to interact with to Swedbank Pay's API Platform in a statically typed client.
```

## Supported APIs

> * PaymentOrder

Reviewer: ```suggestion
* **Payment Order**
```

  * create payment order
  * get payment order
  * capture
  * cancel
  * reversal
  * abort

> * Swish

Reviewer: ```suggestion
* **Swish Payments**
```

  * create swish payment
  * get swish payment
  * abort
  * reversal

> * CreditCard

Reviewer: ```suggestion
* **Card Payments**
```
